example/client/cmd/report.go

```diff
@@ -15,17 +15,77 @@
 package cmd
 
 import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
```

Comment:
Thanks for the review. My local bazel installation is not working. Do you know what this error means:



ozben-macbookpro:mixer ozben$ bazel build ...

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:500:18: expected ConfigurationTransition or NoneType for 'cfg' while calling label but got string instead: host.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:528:20: name 'go_env_attrs' is not defined.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:560:13: Traceback (most recent call last):

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 558

rule(go_library_impl, attrs = go_librar..."])}, <2 more arguments>)

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 560, in rule

go_library_attrs

name 'go_library_attrs' is not defined.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:571:13: Traceback (most recent call last):

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 569

rule(go_binary_impl, attrs = go_library...()}, <3 more arguments>)

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 571, in rule

go_library_attrs

name 'go_library_attrs' is not defined.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:582:13: Traceback (most recent call last):

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 580

rule(go_test_impl, attrs = go_library_a...()}, <4 more arguments>)

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 582, in rule

go_library_attrs

name 'go_library_attrs' is not defined.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:686:13: Traceback (most recent call last):

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 684

rule(_cgo_codegen_impl, attrs = go_env_...)}, <2 more arguments>)

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 686, in rule

go_env_attrs

name 'go_env_attrs' is not defined.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:798:13: Traceback (most recent call last):

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 796

rule(_cgo_import_impl, attrs = go_env_a..."), <2 more arguments>)})

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 798, in rule

go_env_attrs

name 'go_env_attrs' is not defined.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:1044:20: expected ConfigurationTransition or NoneType for 'cfg' while calling label but got string instead: host.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:1055:13: Traceback (most recent call last):

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 1053

repository_rule(implementation = _go_repository_..., ...)

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 1055, in repository_rule

_go_repository_attrs

name '_go_repository_attrs' is not defined.

ERROR: /private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl:1060:13: Traceback (most recent call last):

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 1058

repository_rule(implementation = _new_go_reposit..., ..."), <4 more arguments>)})

File "/private/var/tmp/_bazel_ozben/8f37a417b093a76dc66e776e0ddfb7f1/external/io_bazel_rules_go/go/def.bzl", line 1060, in repository_rule

_go_repository_attrs

name '_go_repository_attrs' is not defined.

ERROR: com.google.devtools.build.lib.packages.BuildFileContainsErrorsException: error loading package '': Extension file 'go/def.bzl' has errors.

INFO: Elapsed time: 0.127s

ozben-macbookpro:mixer ozben$